desktop-app/app/main.dev.js

```diff
@@ -91,7 +91,10 @@ const openWithHandler = filePath => {
     filePath != null &&
     (filePath.endsWith('.html') || filePath.endsWith('.htm'))
   ) {
-    const url = `file://${filePath}`;
+    // code to encode
+    // file URI
+    p1 = encodeURIComponent(filePath);
+    const url = `file://${filePath + p1}`;
```

Comment:
@aUsefulMethod Isn't `filePath + p1` going to repeat the path twice?

Reply:
maybe not work ... encodeURIComponent  encode all of string :D